.github/workflows/reusable-build-extensions-on-linux.yml

```diff
@@ -67,8 +67,27 @@ jobs:
         shell: bash
         run: |
           [ "${{ matrix.plugin }}" == "wasi_nn-openvino-genai" ] && source /root/openvino_genai/setupvars.sh
+          PIPER_INSTALL_PATH=""
+
+          if [[ "${{ matrix.plugin }}" == "wasi_nn-piper" ]]; then
+            echo "--- Building libpiper ---"
+            git clone https://github.com/OHF-Voice/piper1-gpl piper-source
+            cd piper-source/libpiper
+            cmake -Bbuild-deps \
+              -DCMAKE_BUILD_TYPE=Release \
+              -DCMAKE_INSTALL_PREFIX="$PWD/install" \
+              -DBUILD_SHARED_LIBS=OFF \
+              -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+            cmake --build build-deps
+            cmake --install build-deps
+            PIPER_INSTALL_PATH="$PWD/install"
+            echo "--- Verify install ---"
+            ls -R "$PIPER_INSTALL_PATH"
+            cd ../..
```

Comment:
We should move these lines into a standalone script, maybe called `build_libpiper.sh`, and use it instead of exporting everything here.